# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/test/unit/xpcshell.toml
# Commit: d01a1a055fec
# Full Hash: d01a1a055fece12b906b938954f984bd69278ef7
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2025-04-30 20:49:53
# Regressor Bug: 1960824
# File Overlap Count: 1
# Description:
#   Bug 1960824 - Introduce nsICookie::SAMESITE_UNSET, r=edgul,webdriver-reviewers,extension-reviewers,cookie-reviewers,valentin,Sasha,robwu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D246638
# ==============================================================================

diff -r 307ff391dd02 -r d01a1a055fec netwerk/cookie/test/unit/xpcshell.toml
--- a/netwerk/cookie/test/unit/xpcshell.toml	Wed Apr 30 08:53:12 2025 +0000
+++ b/netwerk/cookie/test/unit/xpcshell.toml	Wed Apr 30 08:59:18 2025 +0000
@@ -22,8 +22,6 @@
 
 ["test_parser_0019.js"]
 
-["test_rawSameSite.js"]
-
 ["test_schemeMap.js"]
 
 ["test_timestamp_fixup.js"]